assets/src/apps/delivery/layouts/deck/DeckLayoutView.tsx

```diff
@@ -42,13 +43,40 @@ const InjectedStyles: React.FC<{ css?: string }> = (props) => {
   return injected ? <style>{injected}</style> : null;
 };
 
+const getActualOwnerDetails = (expression: string, sequence: any) => {
+  const [activityId, part] = expression.split('|');
```

Comment:
seems like it's possible there is no `|` for whatever reason, should check for that probably